packages/svelte/README.md

```diff
@@ -278,7 +278,7 @@ To set this up, you simply need to important the vite plugin and add it to your
 
 import { sveltekit } from '@sveltejs/kit/vite';
 import { coverageConfigDefaults, defineConfig } from 'vitest/config';
-+ import { asyncTransform } from '@sheepdog/svelte/lib/vite';
++ import { asyncTransform } from '@sheepdog/svelte/vite';
```

Comment:
Did we need this also in the actual docs?

Reply:
No the actual docs are fine - https://github.com/mainmatter/sheepdog/blob/561ec5100923e6e139edabb67316449bd5929cb5/apps/docs/src/content/docs/getting-started/installation.mdx?plain=1#L38